server/graphql/v2/object/Tier.js

```diff
@@ -64,6 +64,9 @@ export const Tier = new GraphQLObjectType({
           return { value: tier.amount, currency: tier.currency };
         },
       },
+      type: {
+        type: TierType,
```

Comment:
Looks like this should be NonNull, but there's no DB constraint neither

Reply:
Adding that